playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_positioning.html.erb

```diff
@@ -0,0 +1,67 @@
+<%= pb_rails("title", props: {text: "Tokens", size: 4 }) %>
```

Comment:
Pull the latest from master and you will see better examples for spacing for this whole file